### DISCUSSION TITLE
Can you please explain how you have written the code for decrypt_link()

### DISCUSSION THREAD

**Discussion by AvAkanksh** [pystardust/ani-cli#498] [Q&A] [answered]
I was trying to understand how you wrote the code for extracting the video URL.  could you please help me in understanding the code. and why did u use the particular value for

secret_key='3235373436353338353932393338333936373634363632383739383333323838'
iv='34323036393133333738303038313335'

any reason behind it ?

also you u share some resource where I can further understand the implementation of decrypt_link code.

Thanks a lot in advance

**Comment by freezboltz** [comment, marked as answer]
- #217  
- [openssl implementation](https://gist.github.com/alpha-hexor/a639957b054133a7d6198231dd6603f0)
- [main extractor](https://github.com/justfoolingaround/animdl/blob/master/animdl/core/codebase/extractors/gogoplay/__init__.py)
